mysql/tests/conftest.py

```diff
@@ -124,9 +124,7 @@ def _wait_for_it_script():
 
 def _mysql_docker_repo():
     if MYSQL_FLAVOR == 'mysql':
-        if MYSQL_VERSION == '5.5':
-            return 'jfullaondo/mysql-replication'
```

Comment:
Should we leave a comment here stating which image to use for 5.5 or even leave the code in place?

Reply:
@hithwen 👍 Just added this https://github.com/DataDog/integrations-core/pull/4678